backend-api/src/main/java/org/wcci/apimastery/Storage/ArtistRepository.java

```diff
@@ -0,0 +1,13 @@
+package org.wcci.apimastery.Storage;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import org.wcci.apimastery.models.Artist;
+
+import java.util.Optional;
+
+@Repository
+public interface ArtistRepository extends CrudRepository<Artist,Long> {
+    Optional<Artist> findCategoryById(Long id);
```

Comment:
```suggestion
    Optional<Artist> findArtistById(Long id);
```
We don't use categories